docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -80,6 +82,8 @@ kotlin {
         iosMain.dependencies {
             implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesVersion%")
         }
+        linuxX64Main { } // Warning: accessing source set
```

Comment:
I don't think our guidelines are explicit here, but I feel that a multiline comment especially should be placed before the code, not alongside — it also makes it easier to separate changes to the code itself from changes to comments.